sessions/html-forms/demo-end/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>HTML Forms: Demo End</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <h1>🎈 Let's celebrate! 🎁</h1>
+    <h2 id="title">Private Birthday Party Picker</h2>
+    <p id="description">
+      Let's celebrate - it's my birthday! Please give me your name, email, your
+      preferred party date and how many friends you will bring to my birthday
+      party.
+    </p>
+    <form aria-labelledby="title" aria-describedby="description">
```

Comment:
I like the attention to a11y

Reply:
It's part of the session. 😆 I just moved `aria-describedby` from the `fieldset` to the `form` element itself as that makes more sense in my point of view.